yivi-docs/blog/2025-02-08-ux/index.md

```diff
@@ -0,0 +1,104 @@
+
+Understandable means that Yivi should feel familiar and usable to everyone.
+
+Trustworthy is not just about using the most reliable protocols. 
+It's also about conveying that to the user and letting them feel like they can trust Yivi, issuers and verifiers.
+Things that fall into this category are imagery, colors and clear explanations.
+
+Beautiful is not as important as the others, but the eye wants something too.
+Things like consistent color, fonts and styling combined with fluid animations is what gives Yivi just that extra touch.
+
+
+## Our plans for the near future
+There are four UX improvements that we want to work on over the next couple of months.
+These are:
+- Getting compliant with European Accessibility Act
+- QR code scanner button on the login page
+- Easier card organisation and navigation
+- Brand extensions
+
+Let's look at them one by one.
+
+### Getting compliant with European Accessibility Act (EAA)
+The EAA is a directive that aims to improve the functioning of the internal market for accessible products and services,
+by removing barriers created by divergent rules in member states of the European Union.
+
+It applies to products and services that have been identified as being most important to people with disabilities.
+Yivi aims to be one of those products and therefore should comply with this act.
+
+For digital products, like Yivi, this means having to comply with the Web Content Accessibility Guide (WCAG) version 2.1 level AA.
+
+Requirements include:
+- Big enough touch targets
+- Big enough contrast ratios
+- Support for screen readers
+- Alt text for visual media
+- Support for large text
+- And more
+
+We plan to adhere to these standards by assessing our current state and make improvements accordingly.
+Recently we've improved our screen reader support by adding better descriptions and hiding some distracting items from the screen reader.
+
+Automatic tests will be introduced to make sure our Flutter code adhears to the touch target and contrast ratio requirements set by WCAG.
+
+We encourage users to let us know what they think and how we can improve the experience for them.
+
+### QR code scanner button on login page
+For Yivi to be understandable it helps to feel familiar to some degree.
+Lots of banking apps, at least in The Netherlands,
+have a quickly accessible QR code scanner button on the login page that users can press even before entering their pin.
+The pin code then needs to be entered to confirm the action the user wants to take based on the scanned QR code.
+For banking apps that would most likely be transfering money.
+
+Our plan is for Yivi to also support this flow. 
+The user can quickly scan a QR code by pressing the button on the login page 
+and then enter the pin afterwards to go ahead with the issuance or disclosure.
+
+
+### Easier card organisation and navigation
+We've received feedback from some of our users that the navigation and organisation of the cards inside the wallet.
```

Comment:
I think something is missing in this sentence. I feel like it's incomplete. Is that correct?